src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -62,6 +81,10 @@ public class Utils {
   public static final String SF_SSL = "sfssl"; // for test only
   public static final String SF_WAREHOUSE = "sfwarehouse"; // for test only
   public static final String PRIVATE_KEY_PASSPHRASE = "snowflake.private.key" + ".passphrase";
+  public static final String SF_AUTHENTICATOR = "snowflake.authenticator";
```

Comment:
lets rename this to `SF_AUTHENTICATOR_TYPE`

Reply:
what are the allowed values for this config?

Reply:
`snowflake_jwt` or `oauth`

Reply:
Fixed.